code/online_challenges/src/project_euler/problem_005/problem_005.java

```diff
@@ -0,0 +1,19 @@
+public class Problem005 {
+	public static boolean isDivisible(long number) {
+		boolean flag = true;
```

Comment:
Don't use a boolean flag.

Reply:
Why not Boolean ??

Reply:
It is considered bad practice to avoid multiple returns, which will shorten your code and make it easier to understand here.